modes/magit/evil-collection-magit.el

```diff
@@ -1,36 +1,52 @@
-;;; evil-collection-magit.el --- Bindings for `magit' -*- lexical-binding: t -*-
+;;; evil-collection-magit.el --- evil-based key bindings for magit
```

Comment:
Should lexical bindings really be off?

Reply:
The original was off, I believe. I don't want to set it to lexical binding and have to make code changes.